config/crd/kustomization.yaml

```diff
@@ -7,10 +7,6 @@ resources:
 # +kubebuilder:scaffold:crdkustomizeresource
 
 patchesStrategicMerge:
-# patches here are for enabling the conversion webhook for each CRD
-- patches/webhook_in_opentelemetrycollectors.yaml
```

Comment:
please remove only this line and keep the scaffolding metadata in place

Reply:
Done in update